app/src/main/java/woowacourse/shopping/data/repository/DefaultCartRepository.kt

```
@@ -56,9 +57,13 @@ class DefaultCartRepository(
             val pagedCartContents = remoteDataSource.getCartItems(page, PAGE_SIZE)
-            allCartContents.addAll(pagedCartContents)
-            if (pagedCartContents.size < PAGE_SIZE) break
-            page += 1
+            if (pagedCartContents is Result.Success) {
+                allCartContents.addAll(pagedCartContents.data)
+                if (pagedCartContents.data.size < PAGE_SIZE) break
+                page += 1
+                continue
+            }
+            break
         }
```

detekt (1.23.8)

[warning] This loop contains an unconditional jump expression which essentially renders it useless as it will exit the loop during the first iteration.

(detekt.potential-bugs.UnconditionalJumpStatementInLoop)